xml/gl.xml

```diff
@@ -3474,6 +3474,8 @@ typedef unsigned int GLhandleARB;
         <enum value="0x0400" name="GL_SPARSE_STORAGE_BIT_ARB"/>
         <enum value="0x0800" name="GL_LGPU_SEPARATE_STORAGE_BIT_NVX"/>
         <enum value="0x0800" name="GL_PER_GPU_STORAGE_BIT_NV"/>
+        <enum value="0x1000" name="reserved_0x1000_NV"/>
```

Comment:
Use the \<unused start="0x1000" end="0x1000" name="Reserved for NVIDIA"> syntax here until there's an actual enum defined, please.

Reply:
will do, I didn't know about that.

Reply:
done.